README.md

```diff
@@ -5,47 +5,134 @@
 
-To verify that it's working:
+| Topic | Notebook |
+|:--------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| 10-minute tour of Pixeltable    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/pixeltable-basics.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| User-Defined Functions (UDFs)    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/howto/udfs-in-pixeltable.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Comparing Object Detection Models | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/object-detection-in-videos.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Experimenting with Chunking (RAG) | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/rag-operations.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
+| Working with External Files    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/howto/working-with-external-files.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Integrationg with Label Studio for Annotations    | <a target="_blank" href="https://pixeltable.readme.io/docs/label-studio"> <img src="https://img.shields.io/badge/Docs-Label Studio-blue" alt="Visit our documentation"/></a>
 
+## 🧱 Code Samples
+
+### All media data (videos, images, audio) resides in external files, and Pixeltable stores references to those
 ```python
-import pixeltable as pxt
-pxt.init()
+# The files can be local or remote (e.g., in S3). For the latter, Pixeltable automatically caches the files locally on access.
+v = pxt.create_table('external_data.videos', {'video': pxt.VideoType()})
+
+prefix = 's3://multimedia-commons/'
+paths = [
+    'data/videos/mp4/ffe/ffb/ffeffbef41bbc269810b2a1a888de.mp4',
+    'data/videos/mp4/ffe/feb/ffefebb41485539f964760e6115fbc44.mp4',
+    'data/videos/mp4/ffe/f73/ffef7384d698b5f70d411c696247169.mp4'
+]
+v.insert({'video': prefix + p} for p in paths)
 ```
-> [!NOTE]
-> Check out the [Pixeltable Basics](https://pixeltable.readme.io/docs/pixeltable-basics) tutorial for a tour of its most important features.
+Learn more on how to [access the original file paths and work with your data](https://pixeltable.readme.io/docs/working-with-external-files).
```

Comment:
I don't understand what "access the original file paths and work with your data" means